### PR TITLE
Login logout 14 ii

### PR DESCRIPTION
**Pull request by muydanny**
Merchant logged in redirect

Admin redirect spec

admin redirect

Updated ReadMe

User story 16 spec for regular user

Logout spec for regular user

### CODE REVIEW

File: app/views/merch/show.html.erb [hunk 1]
```diff
@@ -1 +1,2 @@
 <h1>Merchant Dashboard</h1>
+
```
Comment (muydanny) [inline app/views/merch/show.html.erb:2]: Cant wait to put everything Mike Dao on here

File: spec/features/sessions/admin_user_new_spec.rb [hunk 2]
```diff
@@ -27,6 +27,28 @@
         expect(page).to_not have_link("Sign up")
         expect(page).to_not have_link("Sign in")
     end
+    it "Users who are logged in already are redirected" do
```
Comment (muydanny) [inline spec/features/sessions/admin_user_new_spec.rb:30]: A lot of these tests are similar but wanted to cover all angles of users

File: spec/features/sessions/admin_user_new_spec.rb [hunk 3]
```diff
@@ -33,8 +33,8 @@
 
       click_on "Sign in"
 
-      fill_in :email, with: @regular_user.email
-      fill_in :password, with: @regular_user.password
+      fill_in :email, with: @admin_1.email
```
Comment (muydanny) [inline spec/features/sessions/admin_user_new_spec.rb:36]: duh

File: app/views/layouts/application.html.erb [hunk 4]
```diff
@@ -23,23 +23,23 @@
           <%= link_to "All Items", "/items"%>
           <%= link_to "Cart: #{cart.total_items}", "/cart" %>
           <%= link_to "Profile", "/profile"%>
-          <%= link_to "Log out", "/logout"%>
+          <%= link_to "Log out", logout_path, method: "delete" %>
```
Comment (muydanny) [inline app/views/layouts/application.html.erb:26]: Check this page out. Its very helpful and gave me a good lead on logout
https://3rd-edition.railstutorial.org/book/log_in_log_out

File: README.md [hunk 5]
```diff
@@ -387,7 +387,7 @@ And I see a link to edit my profile data
 ```
 
 ```
-[ ] done
+[ X ] done
 
 User Story 20, User Can Edit their Profile Data
```
Comment (stellakunzang) [inline README.md:393]: Whoa dude! This is very cool. TAKE THAT README

File: app/views/layouts/application.html.erb (hunk 4 above)
Comment (stellakunzang) [inline app/views/layouts/application.html.erb:26]: ooo lala

**Review by stellakunzang — CHANGES_REQUESTED** [review]
Can you remove the comments of stories that are completed?

**Pull request merged**